tests/unit/test_scale_means.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import division
```

Comment:
is there a test for single-dimensional analysis of scale means?  I know this is probably not a thing, but the users will do it and I think it would be best not to error in this case.

Reply:
There is now. It's not supposed to happen, and we raise explicitly.